data/language/en-GB.txt

```diff
@@ -4121,6 +4121,7 @@ STR_5811    :{SMALLFONT}{BLACK}If checked, you can have up to{NEWLINE}255 cars p
 STR_5812    :Show multiplayer window
 STR_5813    :{OPENQUOTES}{STRING}{ENDQUOTES}
 STR_5814    :{WINDOW_COLOUR_1}{OPENQUOTES}{STRING}{ENDQUOTES}
+STR_5871    :Krone (kr)
```

Comment:
There's a hole section labeled #tooltips, I figured it might be confusing to put it right there. But I could do that.